vnext/Shared/Networking/IHttpResource.h

```diff
@@ -43,9 +89,11 @@ struct IHttpResource {
 
   virtual void ClearCookies() noexcept = 0;
 
-  virtual void SetOnRequest(std::function<void(int64_t requestId)> &&handler) noexcept = 0;
+  virtual void SetOnRequestSuccess(std::function<void(int64_t requestId)> &&handler) noexcept = 0;
   virtual void SetOnResponse(std::function<void(int64_t requestId, Response &&response)> &&handler) noexcept = 0;
   virtual void SetOnData(std::function<void(int64_t requestId, std::string &&responseData)> &&handler) noexcept = 0;
+  virtual void SetOnBlobData(
```

Comment:
>SetOnBlobData [](http://example.com/codeflow?start=15&length=13)

It doesn't have to be blobs, right .. The data depends on whichever urihandler handles is picked, Blob handler could be just one of them ..

Reply:
Agreed. This is not the best name. I'll update.